webClient/src/app/shared/message-duration.ts

```diff
@@ -9,7 +9,7 @@
 */
 
 export class MessageDuration {
-  static Short: number = 2000;
+  static Short: number = 3000;
```

Comment:
Accessibility

Reply:
length is good now, but if multiple generated together first one disappear rather quickly,
but good enhancement